region/maridia/inner-yellow/The Beach.json

```
@@ -779,6 +805,33 @@
       "flashSuitChecked": true,
       "devNote": "This strat is not useful in-room, but can satisfy a strat in the room before with an exit shinespark."
     },
+    {
+      "link": [2, 2],
+      "name": "R-Mode Spark Interrupt (Gain Blue Suit)",
+      "entranceCondition": {
+        "comeInWithRMode": {},
+        "comesThroughToilet": "no"
+      },
+      "requires": [
+        "Gravity",
+        {"or": [
+          "h_CrystalFlashForReserveEnergy",
+          {"and": [
+            "h_RModeCanRefillReserves",
+            {"partialRefill": {"type": "ReserveEnergy", "limit": 20}}
+          ]}
+        ]},
+        "h_shinechargeMaxRunway",
+        {"autoReserveTrigger": {}},
+        "canRModeSparkInterrupt"
+      ],
+      "flashSuitChecked": true,
+      "blueSuitChecked": true,
+      "note": [
+        "Farm the Choots before dropping into the water. Shinecharge along the bottom and use a Skultera",
+        "to interrupt."
```

With speed + gravity it's fine to get up there, so I think the comment is the only thing that needs to be edited
```suggestion
        "Farm the Choots, then shinecharge along the bottom and use a Skultera to interrupt."
```